docs/guides/quick-start/04-document-management.md

```diff
@@ -66,11 +66,14 @@ These documents are associated with order events. The below endpoint creates a d
     "fileType": "pdf",
     "static": false,
     "referencedDocumentId": null,
-    "config": {}
+    "config": {},
+    "sent": true
   }
 }
 ```
 
+While placing an order, during order transaction or order delivery, you have the option to set the `sent` parameter to `true`, which is a boolean flag determining whether to skip documents that have already been marked as sent while processing orders.
```

Comment:
I would split the sentence up for better readability (after `true`). "It is a boolean flag ... "

Possibly add "whether to skip **the creation** of documents that have already..." to be more precise on what is skipped. I thought of **sending** or **generation**.